docs/modules/ROOT/pages/list-of-metrics.adoc

```diff
@@ -92,6 +92,21 @@ execution.
 |Completion time of the current execution of the job (epoch time
 in milliseconds).
 
+|status
+|Information regarding the job's status. Potential values:
+0 - submitted;
+1 - initialization phase;
+2 - running;
+3 - suspended;
+4 - exporting snapshot;
+5 - completing phase;
+6 - failed;
+7 - completed successfully;
+
+|userCancelled
```

Comment:
`userCancelled` name seems to be in UK English (two Ls in cancelled), description in US (single L in canceled).

Not really a docs thing if the UK English is used in the code, but thought I'd mention it....

Reply:
See https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/jet/core/metrics/MetricNames.java#L212

Reply:
I looked through the code and saw that there is no unified approach for the word cancel) so, here you recommend changing the parameter in the code?